### PR TITLE
{cae}[intel/2018a] Code_Saturne 5.x

### PR DESCRIPTION
**Pull request by mcakircali**
It needs [Code_Saturne easyblock](https://github.com/easybuilders/easybuild-easyblocks/pull/1587).

### CODE REVIEW

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('gettext', '0.19.8.1'),
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: Update to `2.7.15`?

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ('PyQt', '4.12.1', '-Python-%(pyver)s'),
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: This version of `PyQt` does not exist in `intel-2018a`. Maybe you missed it in the commit?

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    ('HDF5', '1.8.20'),
+    ('CGNS', '3.3.1'),
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: Same comment as for `PyQt`

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    ('ParMETIS', '4.0.3'),
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: Same comment

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    ('SCOTCH', '6.0.6'),
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: Same comment

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    ('ParaView', '5.4.1', '-OSMesa'),
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: Same comment

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+]
+
+# better to configure these dependents explicitly
+configopts = ['--with-cgns=$EBROOTCGNS --with-metis=$EBROOTPARMETIS --with-scotch=$EBROOTSCOTCH']
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: `configopts` should be a string, not a list, in most cases

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.3.0-intel-2018a.eb [hunk 8]
```diff
@@ -0,0 +1,52 @@
+# easyconfig file for Code_Saturne
+# author: Metin Cakircali (Juelich Supercomputing Centre)
+name = 'Code_Saturne'
+version = '5.3.0'
```
Comment (damianam) [inline easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.3.0-intel-2018a.eb]: Is there a reason to have `5.3.0` and `5.0.9` installed at the same time? If not, I'd suggest to remove `5.0.9` from the PR

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+homepage = 'https://www.code-saturne.org'
+description = """Code_Saturne solves the Navier-Stokes equations 
+for 2D, 2D-axisymmetric and 3D flows, steady or unsteady, 
+laminar or turbulent, incompressible or weakly dilatable, 
+isothermal or not, with scalars transport if required.
+
+Code_Saturne %(version)s%(versionsuffix)s is installed in
+$EBROOTCODE_SATURNE
+"""
+toolchain = {'name': 'intel', 'version': '2018a'}
```
[the remark this answers is not in the document]
Comment (mcakircali) [reply easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: As we talked before, there are some issues regarding paraview. This is intel/2018a config anyway.

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.3.0-intel-2018a.eb (hunk 8 above)
Comment (mcakircali) [reply easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.3.0-intel-2018a.eb]: 5.0.9 is long term support, 5.3.0 is latest..

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb (hunk 7 above)
Comment (mcakircali) [reply easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: okay, not a problem.

File: easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb (hunk 2 above)
Comment (mcakircali) [reply easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: I guess the issue is that I don't have default EB system to test. Any suggestion for that?

Comment (mcakircali) [reply easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: use foss?

Comment (mcakircali) [reply easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb]: add PyQt intel/2018a?

**Comment by boegel** [comment]
Since this is used a deprecated toolchains (see https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains), and the easyblock still needs some work (see https://github.com/easybuilders/easybuild-easyblocks/pull/1587), I'll go ahead and close this PR.

If support for `Code_Saturne` would still be useful, please open a new PR.

**Pull request closed without merging**